js/react/components/auth/login/Login.jsx

```diff
@@ -51,7 +51,7 @@ function Login({ modalRef, dispatch }) {
 					</button>
 				</div>
 			}
-			footer={<p>Agency partner users must log in with work email</p>}
+			footer={<p>Agency partner users must log in with work email.</p>}
```

Comment:
```suggestion
			footer={<p>Agency partners must log in with a work email.</p>}
```

Possible update to the wording here to make it more user-friendly?